### ISSUE TITLE
Advanced Import Settings - Allow saving multiple meshes at once

### ISSUE THREAD

**Issue by nonunknown** [godotengine/godot-proposals#4266]
### Describe the project you are working on

A kart racing game

### Describe the problem or limitation you are having in your project

You have a 3d model with many, many meshes, so you need to save one by one or else the editor will take a long time to save scenes!

![Uploading image.png…]()

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

The enhancement proposed here will make possible to select multiple meshes at once:

![image](https://user-images.githubusercontent.com/58845030/159809762-24270fcf-3d8a-4a3c-8c8b-7e673d1d24c8.png)

and then save all of them in the selected folder:

![image](https://user-images.githubusercontent.com/58845030/159809842-dd1e004e-f84c-4a81-8f3f-e97a9a984ad9.png)

the user can select if he wants tres or res, and also the filename will be the same name of the meshes!

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

described above.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

nope

### Is there a reason why this should be core and not an add-on in the asset library?

its a core feature

**Comment by Calinou** [comment]
@nonunknown One of your images in your proposal post failed to upload, can you try to edit the post and upload it again?

**Comment by nonunknown** [comment]
sorry for the mistake, solved now!

**Comment by lentsius-bark** [comment]
IMO this is a no brainer. It would help speed up and otherwise very tedious process. In my 3D workflow I often like to batch export objects together and then import as separate meshes, which allows me to have Mesh instances in scenes that update automatically when I re-export and re-import the model (with instanced gltf files, one has to first open it after the re-import to see the changes take effect)
In other words imo exporting a batch of objects to import them as meshes is a clever technique and having the ability to batch click this and NOT have to manually input save oaths would be a serious speed up in the given workflow.
